keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -594,18 +596,18 @@ class SqlDeliveryConfigRepository(
           }
         }
         .fetchOne { (
-          deliveryConfigName,
-          environmentName,
-          artifactVersion,
-          artifactReference,
-          constraintType,
-          status,
-          createdAt,
-          judgedBy,
-          judgedAt,
-          comment,
-          attributes
-        ) ->
+                      deliveryConfigName,
+                      environmentName,
+                      artifactVersion,
+                      artifactReference,
+                      constraintType,
+                      status,
+                      createdAt,
+                      judgedBy,
+                      judgedAt,
+                      comment,
+                      attributes
+                    ) ->
```

Comment:
I don't know why spotless did this but I'm not going to waste time fighting it

Reply:
My guess is because of the parenthesis on the same line as the curly brace.